.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
This should be something like https://github.com/mcollina/fastify-sandbox/blob/2c2932895448cda367da53b47be0382cee4f7e27/.github/workflows/ci.yml#L4-L10 or https://github.com/mcollina/fastq/blob/be3943ad8083da97ce7a2660ad8d3e8bbdf2bf4c/.github/workflows/ci.yml#L3